src/Altinn.Profile.Integrations/Migrations/20250304091657_Init.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+using Npgsql.EntityFrameworkCore.PostgreSQL.Metadata;
+
+#nullable disable
+
+namespace Altinn.Profile.Integrations.Migrations
+{
+    /// <inheritdoc />
+    public partial class Init : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
```

Comment:
Manually deleted the content of these files as we don't want `up `or `down `to touch the existing database

Reply:
If we want to use yuniql now, these will be deleted

Reply:
Let's stick with yuniql for now